async_fuse/src/aligned_bytes.rs

```diff
@@ -16,11 +13,14 @@ use core::ops::{Deref, DerefMut};
 use core::ptr::NonNull;
 use core::slice;
 use std::alloc::{self as alloc_api, Layout};
+use utilities::OverflowArithmetic;
```

Comment:
The version of `utilities` is not locked.
1. Check `Cargo.lock` into `git`
2. Specify the branch and commit of `utilities`

Reply:
specify branch and commit
`utilities = { git = "https://github.com/datenlord/utilities", branch = "master", rev = "044ae02" }`